### PR TITLE
Support tidb-binlog metrics scraping for TidbMonitor

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Support tidb-binlog metrics scraping for TidbMonitor like below:

image

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
Support scraping `Pump` and `Drainer` metrics in TidbMonitor
```

### CODE REVIEW

**Comment by Yisaer** [comment]
@tirsen PTAL

File: pkg/monitor/monitor/template.go [hunk 1]
```diff
@@ -126,6 +138,8 @@ func newPrometheusConfig(cmodel *MonitorConfigModel) *config.Config {
 			scrapeJob("tikv", tikvPattern, cmodel, buildAddressRelabelConfig(portLabel, tikvReplacement, true)),
 			scrapeJob("tiflash", tiflashPattern, cmodel, buildAddressRelabelConfig(portLabel, tiflashReplacement, true)),
 			scrapeJob("tiflash-proxy", tiflashPattern, cmodel, buildAddressRelabelConfig(tiflashProxyPortLabel, tiflashProxyReplacement, true)),
+			scrapeJob("pump", pumpPattern, cmodel, buildAddressRelabelConfig(portLabel, pumpReplacement, true)),
```
Comment (DanielZhangQD) [inline pkg/monitor/monitor/template.go:141]: Does the function in L286 for TLS need to be updated?

Comment (Yisaer) [reply pkg/monitor/monitor/template.go:141]: updated.

**Comment by DanielZhangQD** [comment]
Please make sure the monitoring dashboard works as expected for clusters with or without TLS enabled.

**Pull request merged**